### PR TITLE
project: update dependencies

### PR DESCRIPTION
**Pull request by rerowep**
* Uses flask_babel instead flask_babelex.

### CODE REVIEW

**Review by Garfield-fr — APPROVED** [review]

**Review by PascalRepond — CHANGES_REQUESTED** [review]
No pyproject change?
Tested and this doesn't work.

File: examples/app.py [hunk 1]
```diff
@@ -19,13 +18,24 @@
...
     app.config.from_mapping(
         SECRET_KEY='dev',
-        WIKI_CURRENT_LANGUAGE=lambda: session.get('ln', 'fr'),
-        WIKI_LANGUAGES=['en', 'fr', 'de', 'it'],
-        # BABEL_TRANSLATION_DIRECTORIES = resource_filename('flask_wiki', 'translations'),
-        BABEL_DEFAULT_LOCALE='en',
+        WIKI_CURRENT_LANGUAGE=lambda: session.get('language', 'fr'),
+        WIKI_LANGUAGES={
+            'en': 'English',
+            'fr': 'Frensh',
```
Comment (PascalRepond) [inline examples/app.py]: French*

**Review by PascalRepond — APPROVED** [review]

File: examples/app.py [hunk 2]
```diff
@@ -19,13 +18,24 @@
 
 def create_app(test_config=None):
     # create and configure the app
+    def get_locale():
```
Comment (jma) [inline examples/app.py:21]: Is `WIKI_LANGUAGES` not used anymore?

File: examples/data/files/.DS_Store
Comment (jma) [inline examples/data/files/.DS_Store:1]: Please remove this and add this kind of files in the .gitignore.

File: flask_wiki/.DS_Store
Comment (jma) [inline flask_wiki/.DS_Store:1]: Same here.

File: flask_wiki/templates/wiki/base.html [hunk 3]
```diff
@@ -63,12 +63,16 @@
...
     <div class="row">
       <div class="col-md-6">
-        <small class="d-block mb-3 text-muted">RERO &copy; 2020</small>
+        <small class="d-block mb-3 text-muted">RERO &copy; 2023</small>
       </div>
       <div class="col-md-6">
-        {% for language in g._wiki.languages -%}
-        <a class="ml-2" href="">{{ language }}</a>
-        {%- endfor %}
+        {% for language in AVAILABLE_LANGUAGES.items() %}
```
Comment (jma) [inline flask_wiki/templates/wiki/base.html]: Why not using `g`?

File: flask_wiki/translations/de/LC_MESSAGES/messages.mo
Comment (jma) [inline flask_wiki/translations/de/LC_MESSAGES/messages.mo:1]: Please remove this from git and add a .gitignore.

File: flask_wiki/translations/hu/LC_MESSAGES/messages.po
Comment (jma) [inline flask_wiki/translations/hu/LC_MESSAGES/messages.po:1]: Regenerate translations seems useless.

**Review by jma — CHANGES_REQUESTED** [review]

File: examples/app.py (hunk 2 above)
Comment (rerowep) [reply examples/app.py:21]: I don't know, why do you ask?

**Review by jma — APPROVED** [review]

**Pull request merged**